examples/docs_beta_snippets/README.md

```diff
@@ -3,6 +3,21 @@
 This module exists only to enable testing docs snippets/examples in CI, and should not be installed
 otherwise.
 
+## Code Style
+
+Please use the following code style to keep imports short:
+
+```python
+import dagster as dg
+
```

Comment:
Shouldn't this function be decorated with `@dg.asset`?

Reply:
it should :) feel free to throw up a pr if you want in on the action

Reply:
```suggestion
@dg.asset
```

Reply:
I think this should work 😅

Reply:
Hey @PedramNavid! Does the GitHub suggestion works or should I send a PR? Happy to do it but wanted to try the suggestion way first.